app/controllers/campaigns_controller.rb

```diff
@@ -22,6 +22,11 @@ def show
   # POST /campaigns
   def create
     @campaign = Campaign.create!(create_params)
+    if params[:has_square_fee]
+      @fee = Fee.find(1)
```

Comment:
small suggestion: move the `1` ("magic number"!) into a const like "SQUARE_FEE_ID" so that it is descriptive. (But might be okay to ignore for now since I guess we only have one Fee on prod lol)